src/third-party-tags/twitter-uwt.ts

```diff
@@ -1,8 +1,8 @@
 import { GetThirdPartyTag } from '../types';
-import { twitterScript } from './external-scripts/twitter-script';
+import { twitterScript as insertSnippet } from './external-scripts/twitter-script';
 
 export const twitter: GetThirdPartyTag = ({ shouldRun }) => ({
 	shouldRun,
 	name: 'twitter',
-	insertSnippet: twitterScript,
+	insertSnippet: insertSnippet,
```

Comment:
this is even shorter!

```suggestion
	insertSnippet,
```